test/acceptance/integration/sth/steps.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of Short Term Historic (FI-WARE project).
+#
+# iot-sth is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+# iot-sth is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with iot-sth. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact:
+#  iot_support at tid.es
+#
+__author__ = 'Iván Arias León (ivan.ariasleon at telefonica dot com)'
+
+from lettuce import step, world
+
+from tools.properties_config import Properties
+
+@step (u'update properties.json file from "([^"]*)" and sudo local "([^"]*)"')
+def copy_properties_file_from_and_sudo(step, file_name, sudo_run):
+    """
+    copy properties.json specific to feature from setting folder, read properties and create necessaries class
+    :param step:
+    :param file_name: file name of configuration associated at features, stored in settings folder
+    :param sudo_run:  with superuser privileges (True | False)
+    """
+    if not world.background_executed:
+        properties = Properties (file=file_name, sudo=sudo_run)
+        properties.read_properties()
+        properties.storing_dictionaries()
+
+@step (u'restart sth service. This execution is only once "([^"]*)"')
+def restart_sth_service_this_execution_is_only_once(step, only_once):
+    """
+    restart sth service
+    :param only_once: determine if the configuration is execute only once or no (True | False)
+    :param step:
+    """
+    if not world.background_executed:
+        world.sth.sth_service("restart")
+    if only_once.lower() == "true":
+        world.background_executed = True
+    else:
+         world.background_executed = False
+
+@step (u'verify if sth is installed correctly')
+def verify_if_sth_is_installed_correctly(step):
+    """
+    verify if sth is installed correctly
+    :param step:
+    """
+    world.sth.verify_sth_version()
+
+@step (u'mongo is installed correctly')
+def mongo_is_installed_correctly(step):
+    """
+    mongo is installed correctly, main verify mongo version (see properties.json)
+    :param step:
+    """
+    world.mongo.connect()
+    world.mongo.eval_version()
```

Comment:
Same here, if you are going to check the installation, put an assert in the step implementation